_posts/2015-03-14-Hi-this-is-oscillatingworks.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: Hi, this is oscillatingworks!
+date: 2015-03-14 00:00:00
+description: We are thrilled to announce our existence, so hi, here we are!
+author: oscillatingworks
+hn-discussion:
+---
+
+Hi everyone!
+
+This is **[oscillatingworks]**, and we love open source. Our mission is to find
+exciting independent, homemade open source software projects, work hand by hand
```

Comment:
I'll go with `exciting, independent, and homemade open source software projects`